### ISSUE TITLE
[woff2] Support hmtx transformation

### ISSUE THREAD

**Issue by khaledhosny** [fonttools/fonttools#650]
The spec has been updated since WOFF2 support was added to FontTools, one of the additions is hmtx transformations. Processing such a font with FonTools gives:

```
$ ttx valid-005.woff2
Dumping "valid-005.woff2" to "valid-005.ttx"...
ERROR: bits 6-7 are reserved and must be 0
```

[valid-005.woff2.zip](https://github.com/behdad/fonttools/files/409108/valid-005.woff2.zip)

**Comment by anthrotype** [comment]
I know.. I still haven't found the time to add that.
Maybe you could send a PR if you need it soon?

**Comment by khaledhosny** [comment]
I don’t have a pressing need, but I’ll give it a try if I got time.

**Comment by anthrotype** [comment]
@khaledhosny Does the google/woff2 command line tool have an option to make hmtx transform, or is it only available in the library?

**Comment by khaledhosny** [comment]
Seems not, the `TransformHmtxTable` function is not used any where. cc @rsheeter

**Comment by anthrotype** [comment]
ok, thanks for checking

**Comment by rsheeter** [comment]
Hm. I was about to say it was but evidently the calling path is stripped during open source prep. google/woff2#52 :)

**Comment by anthrotype** [comment]
Fixed by #1639
